utils/create-icons-map.js

```diff
@@ -0,0 +1,41 @@
+const fs = require('fs');
+const path = require('path');
+
+const svgFolder = '../assets/icons/';
+const iconMapFile = '../src/iconsMap.js';
+
+let iconsObject = '';
+
+let iconMapDir = path.dirname(iconMapFile);
+if (!fs.existsSync(iconMapDir)) {
+    fs.mkdirSync(iconMapDir);
+};
+
+
+
+fs.readdirSync(svgFolder).forEach(file => {
+    let fileName = file.split('.')[0];
+    let filePath = svgFolder + file;
+    if (fs.existsSync(filePath)) {
+        const iconBody = fs.readFileSync(filePath).toString();
+        const cleanedIcon = preparIcon(iconBody, fileName);
+        iconsObject += "'" + fileName + '\' : \'' + cleanedIcon + '\', \n';
```

Comment:
What if `cleanedIcon` includes an unescaped character for a string, e.g `'` or `\` ?